README.md

```diff
@@ -89,6 +89,9 @@ With more options:
 user2 = Plaid::User.exchange_token('public_token', account_id: '...', product: :auth)
 ```
 
+If you specify the `account_id` param, the returned user instance will have the
+`stripe_bank_account_token` attribute set.
+
```

Comment:
It's described here: https://plaid.com/docs/link/stripe/#step-4-write-server-side-handler

@michaelckelly anything to add?